pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,38 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
```

Comment:
## FOR TEMPLATE UPDATE OF RULES

When updating rules to the template it seems that the format "wcag20:x.x.x" breaks the table view in the preview, whereas "wcag20: x.x.x" (with space) doesn't. 
Which one should we go for?